app/views/account/admin/categories/_category_item.html.slim

```diff
@@ -0,0 +1,22 @@
+td.col-md-2.col-sm-3.col-xs-12.flex-order-0
+  span.user-label Створено:
+  = object.created_at.strftime("%m.%d.%Y, %T")
```

Comment:
I saw that in few places you use `strftime("%m.%d.%Y, %T")` so maybe it is reasonable move it to halper